docs/naaccr_data.xsd

```diff
@@ -1,21 +1,26 @@
-<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://naaccr.org/naaccrdata" targetNamespace="http://naaccr.org/naaccrdata" >
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://naaccr.org/naaccrxml" targetNamespace="http://naaccr.org/naaccrxml" >
 
-<xsd:element name="NaaccrData">
+<xsd:simpleType name="recordtypeType">
```

Comment:
<!-- DIV {margin:0px;} -->Nope, this is recordtypeType, not recordType :)  Restricts the attribute to permissible values.
-----Original Message----- From: Fabian Depry Sent: Feb 25, 2015 3:13 PM To: depryf/naaccr-xml Cc: Roger Friedman Subject: Re: [naaccr-xml] Post-Atlanta meeting (#5) > > -> +This should be "recordType" :-)---Reply to this email directly or view it on GitHub:https://github.com/depryf/naaccr-xml/pull/5/files#r25374507